api/src/pcapi/alembic/run_migrations.py

```diff
@@ -13,7 +13,7 @@
 
 target_metadata = db.metadata
 
-IGNORED_COLUMNS_BY_TABLE = {"booking": ("isUsed", "isCancelled")}
+IGNORED_COLUMNS_BY_TABLE = {"booking": ("isUsed", "isCancelled", "isBeneficiary")}
```

Comment:
C'est pas la bonne table 😅 
```suggestion
IGNORED_COLUMNS_BY_TABLE = {"booking": ("isUsed", "isCancelled"}, "user": ( "isBeneficiary")}
```

Tu peux tester fonctionnellement en lançant une migration :)

Reply:
Ah oui mince! J'ai été trop rapide
En effet j'ai testé que tout se lançait bien en ayant supprimé la colonne, mais pas l'auto-migration...